index.html

```diff
@@ -518,27 +517,25 @@ <h2>Protected Header Parameters</h2>
         apply to securing claims about a credential subject.
       </p>
       <p>
-        When replicating claims from the claimset to the header, it is
-        RECOMMENDED to use [[RFC7519]], <a href="https://www.iana.org/assignments/jose/jose.xhtml">IANA
-          Assignments for Header Parameters</a>, and <a href="https://www.iana.org/assignments/jwt/jwt.xhtml">IANA
-          Assignments for JSON Web Token (JWT)</a>
-        to identify any reserved claims that might be confused with
-        members of the [[VC-DATA-MODEL]. This includes but is not
+        When replicating claims from the JWT Claims Set to Header Parameters, it is
+        RECOMMENDED to use [[RFC7519]],
+	the IANA <a href="https://www.iana.org/assignments/jwt/jwt.xhtml">JSON Web Token Claims</a> registry, or
+	the IANA <a href="https://www.iana.org/assignments/jose/jose.xhtml">JSON Web Signature and Encryption Header Parameters</a> registry
+        to identify any claims that might be confused with
+        members defined by the [[VC-DATA-MODEL]. This includes but is not
```

Comment:
```suggestion
        members defined by the [[VC-DATA-MODEL]. These include but are not
```

Reply:
`any claims that might be confused with members` is effectively plural, as is the list of included claims. `These` is then the appropriate referring pronoun, and `are not` is then the appropriate verb.

-----

It does NOT make me happy when my change requests are closed without even the courtesy of a comment, especially when what remains is simply wrong. Please don't do that.

Reply:
@TallTed I applied all of your proposed changed to the PR.  Thank you for suggesting them.